### ISSUE TITLE
voxel sizes wrong

### ISSUE THREAD

**Issue by tischi** [mobie/mobie-viewer-fiji#506]
@KateMoreva @K-Meech @constantinpape 
Now that I pulled the latest develop branch the issue with the wrong voxel sizes in BDV is back. @KateMoreva could you please look into it? 

For testing:

```
		final ImageJ imageJ = new ImageJ();
		imageJ.ui().showUI();
		new MoBIE( "https://github.com/mobie/platybrowser-datasets", MoBIESettings.settings().gitProjectBranch( "spec-v2" ) );
```

The voxel sizes are `px` instead of `micrometer`.

<img width="300" alt="image" src="https://user-images.githubusercontent.com/2157566/140103062-5d0a00cb-fdb0-429a-9fd9-efdb6e56a53c.png">

**Comment by tischi** [comment]
@KateMoreva 

I started looking a little bit and I think it could be in:
`AbstractSequenceDescription` where there is `BasicViewSetup`, which holds the `VoxelDimensions`. I guess you somehow have to populate the `BasicViewSetup` from the `xml` with the `VoxelDimensions`, which are one of the xml tags.

**Comment by KateMoreva** [comment]
I fixed this issue in mobie-io 1.0.6

**Comment by tischi** [comment]
Thanks a lot!!
